plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -149,7 +150,9 @@ export const Stepper = (stepperProps: StepperProps) => {
     props.onCreate(formState);
     const name =
       typeof formState.name === 'string' ? formState.name : undefined;
-    analytics.captureEvent('create', name ?? props.templateName ?? 'unknown');
+    analytics.captureEvent('create', name ?? props.templateName ?? 'unknown', {
+      value: props.minutesSaved,
```

Comment:
Is it ok that the property is called `value`?  Should this sit under `attributes` or `context`?

Reply:
Yeah, it's [a core part](https://backstage.io/docs/plugins/analytics/#providing-extra-attributes) of an analytics event.  Every event can have one numeric value associated with it, typically representing the "primary" metric one might want to know for a given action.  Existing uses of this include...

- `search` - The number of results returned.
- `discover` - The "rank" of the search result that was clicked on.

This is proposing that the "primary" metric to capture on `create` events is the "amount of time saved."

Reply:
I'm kinda worried this could get misunderstood for something else, e.g. "Time it took to run" or "Index of the run" or  ..etc. Is it ok that we use the _only_ value property to represent this?